datasources/datasources.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+datasources:
+  - name: Prometheus
+    type: prometheus
+    url: http://kps-prometheus.glueops-core-kube-prometheus-stack:9090/
```

Comment:
We shouldn't need this if the local metrics are also going to cortex, right?

Reply:
That's right. Just added this for testing and can be removed.

Reply:
Perfect. Let's pull it out.